internal/oem/oem.go

```diff
@@ -110,6 +107,18 @@ func init() {
 		defaultUserConfig: types.Config{Systemd: types.Systemd{Units: []types.Unit{userCloudInit("OpenStack", "ec2-compat")}}},
 	})
 	configs.Register(Config{
+		name:  "cloudstack",
+		fetch: cloudstack.FetchConfig,
+		baseConfig: types.Config{
+			Systemd: types.Systemd{
+				Units: []types.Unit{
+					{Enable: true, Name: "coreos-metadata-sshkeys@.service"},
+					{Enable: true, Name: "coreos-metadata.service"},
```

Comment:
We generally don't want to enable this service by default. If a particular service needs metadata, it will pull it in directly.